packages/bpk-component-boilerplate/readme.md

```diff
@@ -0,0 +1,31 @@
+# bpk-component-boilerplate
+
+> Backpack example component.
+
+## Installation
+
+```sh
+npm install bpk-component-boilerplate --save-dev
+```
+
+## Usage
+
+```js
+import React from 'react';
+import BpkBoilerplate from 'bpk-component-code';
+
+export default () => (
+  <div>
+    <BpkBoilerPlate title="Hello world" />
+    <BpkBoilerPlate title="Hello world" alternate />
+  </div>
+);
+```
+
+## Props
+
+| Property  | PropType | Required | Default Value |
+| --------- | -------- | -------- | ------------- |
+| title     | Node     | true     | -             |
```

Comment:
I can see the values of having those props to show how to document them, but I think it'd be better to just remove them (except className)  and leave a TODO here. e.g. `TODO: document propTypes - required props first in alphabetical order then optional`